uat/testing-features/src/main/resources/greengrass/features/log-manager-1.feature

```diff
@@ -56,80 +55,51 @@ Feature: Greengrass V2 LogManager
         And I verify that it created a log group of type UserComponent for component UserComponentM, with streams within 60 seconds in CloudWatch
         And I verify 20 logs for UserComponentM of type UserComponent have been uploaded to Cloudwatch within 60 seconds
 
-    Scenario: LogManager-1-T1-b: As a customer I can configure the logs uploader component using a componentLogsConfigurationMap and logs are uploaded to CloudWatch
+    Scenario: LogManager-1-T2: Files are deleted after successfully being uploaded to cloudwatch
+        Given I create a log directory for component UserComponentX
         Given I create a Greengrass deployment with components
             | aws.greengrass.Cli        | LATEST                                    |
             | aws.greengrass.LogManager | classpath:/greengrass/recipes/recipe.yaml |
-        And 5 temporary rotated log files for component aws.greengrass.Nucleus have been created
```

Comment:
Deleted this test because it was the exact same one as the one above with the exception of testing an old configuration key that was used before

Reply:
I don't think we should delete this test unless we've at least one test already using these keys.

Reply:
I disagree. Not everything should be tested with UATs. We already have integration and unit tests covering this. It is just redundant and not providing any additional level of confidence that a code change would break the functionality

Reply:
We deprecated `componentLogsConfiguration` for map. There should be a test with `componentLogsConfiguration` as there can be a customer out there using that key.

Reply:
Yeah. Still, look at the test, that is not what this one is testing. We could ad a new one testing that but this one isn't testing that it is just duplicating the same above. We have integration tests that cover working with that old configuration

Reply:
I can create a follow up one with just that configuration

Reply:
My comment may not be relevant for this change in the PR. Just calling that out in general.